### ISSUE TITLE
Migration strategy for having changes to analytics table updated in projects already using the gem

### ISSUE THREAD

**Issue by akshaymohite** [dtreelabs/specterr#32]
Conversation from Slack.

**Akshay** [3:50 PM]
- @here currently, we create required table *specterr_analytics* on the fly.
- if it exists, we don’t.
- what should we do if columns are changed?
- for e.g. let’s say we want to remove `line_no` column

**Kapil** [3:52 PM]
- I think, in create table we have to add it. by dropping existing tables.

**Akshay** [3:52 PM]
- for now, it’s fine
- in future, let’s say people already have this table
- in short, we don’t have migration strategy as of now

We need to come up with a migration strategy for having changes to analytics table updated in projects already using the gem.
